erpnext/stock/doctype/delivery_note/delivery_note.json

```diff
@@ -1099,7 +1099,7 @@
    "no_copy": 1,
    "oldfieldname": "status",
    "oldfieldtype": "Select",
-   "options": "\nDraft\nTo Bill\nTo Deliver\nOut for Delivery\nDelivered\nCompleted\nCancelled\nClosed",
+   "options": "\nDraft\nTo Bill\nTo Deliver\nIn Transit\nDelivered\nCompleted\nCancelled\nClosed",
```

Comment:
If you're changing an existing status, you'll have to write a patch to set them to the new status